src/javaScript/indexDom.js

```diff
@@ -60,24 +60,24 @@ document.addEventListener('DOMContentLoaded', () => {
   })
 
   function filterAndDisplayGames (input) {
-    const games = document.querySelectorAll('[data-name]')
     let visibleGamesCount = 0
+    const gridContainer = document.querySelectorAll('.grid-container')
 
-    games.forEach((game) => {
+    gridContainer.forEach((game) => {
       const name = game.getAttribute('data-name').toLowerCase()
 
-      if (name.includes(input)) {
-        game.style.display = 'block'
-        visibleGamesCount++
-      } else {
-        game.style.display = 'none'
+      if (game.getAttribute('data-name').toLowerCase() === name) {
```

Comment:
Is name always lowercase as well? 
Maybe force it so you don't depend on the user input as much